modules/farm_fd2_school/src/entrypoints/FD1/App.vue

```diff
@@ -176,6 +171,30 @@ export default {
       this.unit = null;
       this.comment = '';
     },
+    async fetchCrops() {
+      const cropsRaw = await fetch(
+        'https://farmos/api/taxonomy_term/plant_type'
+      );
+      const fetchedCrops = await cropsRaw.json();
+      this.cropList = fetchedCrops.data;
+    },
+    async fetchPlantList(cropURL) {
+      const cropListRaw = await fetch(cropURL);
+      const cropListFetched = await cropListRaw.json();
+      if (!Array.isArray(cropListFetched)) this.plantList = [];
+      else this.plantList = cropListFetched;
+    },
+  },
+  watch: {
+    crop() {
+      if (this.crop == null) return null;
+      this.fetchPlantList(
```

Comment:
- The method that you added to fetch the plants is `async`. So it should be called using `await` in your `watch`. 
  - This probably seems to work for now without `await`, but it creates a race condition that will be likely source of bugs in the future.